### DISCUSSION TITLE
Remember 'Clear Cache' options

### DISCUSSION THREAD

**Discussion by lukeyouell** [craftcms/cms#8514] [Ideas]
### Description

It would be nice if the 'Clear Cache' checkbox options were remembered, instead of having to de-select and then re-select again. What do you think?

**Comment by davidhellmann** [comment]
Oh that sounds good. No imager cleaning needed all the time 😀

**Comment by hiasl** [comment]
+1 (no imager cleaning, no Blitz, ...)

**Comment by mmikkel** [comment]
The [CP Clear Cache](https://plugins.craftcms.com/cp-clearcache) plugin does this, and I'd love to see the native clear caches utility work the same way. In fact, just like with the plugin I'd love it if Craft put a shortcut to the utility somewhere too (CP Clear Cache adds a button to the main menu), but I'm sure there are different opinions about that.
